SampleApps/AEPCommerceDemoApp/Podfile

```diff
@@ -7,13 +7,14 @@ project 'AEPCommerceDemoApp.xcodeproj'
 target 'AEPCommerceDemoApp' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
-  pod 'AEPExperiencePlatform', :git => 'git@github.com:adobe/aepsdk-platform-ios.git', :branch => 'dev'
+  pod 'AEPEdge', :path => '../..'
```

Comment:
Can we keep referencing it from git here?

Reply:
Not in this PR, since the AEPEdge podspec doesn't exist yet CI will fail to build the sample app. I can swap this back on the follow-up PR.